generator/internal/scaffold_generator.cc

```diff
@@ -98,7 +98,11 @@ std::map<std::string, std::string> ScaffoldVars(
     if (!api.contains("directory")) continue;
     auto const directory = api["directory"].get<std::string>() + "/";
     if (service.service_proto_path().rfind(directory, 0) != 0) continue;
-    vars.emplace("title", api.value("title", ""));
+    // If the title ends in " API", strip the ending.
+    auto title = api.value("title", "");
+    auto pos = title.size() - std::strlen(" API");
+    if (pos > 0 && title.substr(pos) == " API") title = title.substr(0, pos);
```

Comment:
I think you're risking `unsigned`-arithmetic problems there.  Consider using `absl::EndsWith()`.

Reply:
Ah, that's the one.... thanks, I don't know why I wrote this myself 🤦‍♂️.

(Not fixed yet. Will wait to see if we even want this change)